src/common/m_helper_basic.fpp

```diff
@@ -142,6 +143,10 @@ contains
             buff_size = max(buff_size, 6)
         end if
 
+        if (ib) then
+            buff_size = max(buff_size, 10)
```

Comment:
Does this depend at all on the grid resolution, the size of the IBM compared to the grid resolution, or if an STL file is used? If so, we could use that as a heuristic for how large the lower bound should be.